R/validations.R

```diff
@@ -111,13 +113,14 @@ validate_one_row_per_id <- function(x, key = c("USUBJID", "STUDYID")) {
 
 #' Validates that vector includes all expected values
 #'
-#' @description `r lifecycle::badge("stable")`
+#' `r lifecycle::badge("stable")`
```

Comment:
Won't this push the second paragraph to `@details` section?

```suggestion
#' @description `r lifecycle::badge("stable")`
```